react_ujs/index.js

```diff
@@ -6,6 +6,9 @@ var detectEvents = require("./src/events/detect")
 var constructorFromGlobal = require("./src/getConstructor/fromGlobal")
 var constructorFromRequireContextWithGlobalFallback = require("./src/getConstructor/fromRequireContextWithGlobalFallback")
 
+var renderWithReactDOM = require("./src/renderComponent/withReactDOM")
+var renderWithHotReload = require("./src/renderComponent/withHotReload")
```

Comment:
As mentioned in the [add render component extension point PR](https://github.com/reactjs/react-rails/pull/1064#discussion_r415173047), the folder structure and import pattern is taking reference from getConstructor for consistency. Please let me know if there is a better way